.github/workflows/CreateRelease.ps1

```diff
@@ -0,0 +1,95 @@
+# Required parameter specifying the secret access token to view releases and manage approvals.
+param([Parameter(Mandatory=$true)][string]$accessToken)
+
+# Function to help log out useful information.
+function log ([parameter(mandatory)][String]$msg) { Add-Content log.txt $msg }
+
+# Generate a log file.
+New-Item ./log.txt -ItemType File -Force
+
+# Releases are only created every other week even though the action occurs every week.
+# Identify if this is the second week of a sprint.
+$currentDate = Get-Date
+$currentDate = $currentDate.ToUniversalTime().Date
+$firstRun = ([datetime]::ParseExact('10-23-2020', 'MM-dd-yyyy', $null)).Date
+$weeksSince = [math]::floor(($currentDate - $firstRun).TotalDays / 7)
+$shouldRelease = ($weeksSince % 2) -eq 0
+
+if($shouldRelease) {
+    log "We're not releasing this week, one week left!"
+    exit
+}
+
+# Set basis to call AzureDevOps' Rest API as necessary.
+[Net.ServicePointManager]::SecurityProtocol =  [Net.SecurityProtocolType]::Tls12
+$azureDevOpsAuthenicationHeader = @{Authorization = 'Basic ' + [Convert]::ToBase64String([Text.Encoding]::ASCII.GetBytes(":$($accessToken)")) }
+$apiBase = "https://vsrm.dev.azure.com/microsofthealthoss"
+
+# Hardcoded to point to Release Defenition 23 (DICOM Server Nuget and Tag Release) under the FhirServer Project
+$definitionUri = $apiBase + "/7621b231-1a7d-4364-935b-2f72b911c43d/_apis/Release/definitions/23"
+
+# Identify the latest release
+$definition = Invoke-RestMethod -Uri $definitionUri -Method get -Headers $azureDevOpsAuthenicationHeader
+$lastRelease = $definition.lastRelease.Id
+$currentRelease = $null
+
+while(-Not($lastRelease -eq $currentRelease)) {
+    # Fetch releases currently pending for approval
+    log "Fetching approvals"
+    $approvalsUri = $apiBase + "/fhirserver/_apis/release/approvals?api-version=6.0"
+    $approvals = Invoke-RestMethod -Uri $approvalsUri -Method get -Headers $azureDevOpsAuthenicationHeader
+    $approval = @()
+    foreach($instance in $approvals.value) {
+        # Isolate the one associated with dicom-server
+        if ($instance.releaseDefinition.id -eq "23") { 
+            $approval += $instance
+        }
+    }
+
+    # Validate that only one release is pending approval - any others should be approved, rejected or queued.
+    if($approval.Count -gt 1) {
+        throw "Error: More than 1 approval at a time was unexpected"
```

Comment:
Does this log or just show as a failed message? Should we log the pending approvals?

Reply:
added a log statement with pending approvals.